the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js

```diff
@@ -103,9 +104,13 @@ const JoinRoom = () => {
 
   const handleSelectChange = (event) => {
     let value = event.target.value;
+    setTotalPlayers(value);
     const response = axios.post(IP + `user/play/${value}/${id_room}`);
   };
 
+  const [totalPlayers, setTotalPlayers] = useState(4);
+  const [currentPlayersNumber, setCurrentPlayersNumber] = useState(1);
```

Comment:
These two should have same naming format.
Total could be Max to be more precise.